easybuild/easyconfigs/c/CGAL/CGAL-4.5-intel-2014b-Python-2.7.8.eb

```diff
@@ -0,0 +1,32 @@
+name = 'CGAL'
+version = '4.5'
+
+homepage = 'http://www.cgal.org/'
+description = """The goal of the CGAL Open Source Project is to provide easy access to efficient 
+ and reliable geometric algorithms in the form of a C++ library."""
+
+
+toolchain = {'name': 'intel', 'version': '2014b'}
+toolchainopts = {'strict': True}
+
+sources = [SOURCE_TAR_BZ2]
+source_urls = ['https://gforge.inria.fr/frs/download.php/file/34150/']
+# The directory changes with the version (and maybe in time) ---^
+# Have to accept license agreement
```

Comment:
Well, if the version is bumped, the dowload will fails during testing, so I don't think it has a lot of value?

If you insist on including it, put it right above `source_urls`